### ISSUE TITLE
ButtonSetWithOverflow causing an error in react-resize-detector

### ISSUE THREAD

**Issue by chrisbygrave** [carbon-design-system/ibm-cloud-cognitive#1740]
Found when using PageHeader with pageActions provided on all recent versions of the package.

Shortly after initial render, the component crashes with an error:
```
TypeError
Argument 1 ('target') to ResizeObserver.observe must be an instance of Element
```

The simplest recreate is to access the component in your own code sandbox:
https://codesandbox.io/s/github/carbon-design-system/ibm-cloud-cognitive/tree/main/examples/carbon-for-ibm-products/PageHeader?file=/src/components/SamplePage/SamplePage.js:776-787

Accessing this today in Safari results in the above error.

Removing the `pageActions` property of PageHeader avoids the issue. It also means that no ButtonSetWithOverflow is rendered, which is where I think the problem ultimately lies.

**Comment by shidunce** [comment]
Having same issue. Will be glad to have it fixes in some future version.

**Comment by lee-chase** [comment]
This seems to have been resolved in 1.10, please re-open if still seeing this issue.
